Formula/gromacs.rb

```diff
@@ -54,6 +54,7 @@ def install
     args = %W[
       -DGROMACS_CXX_COMPILER=#{cxx}
       -DGMX_VERSION_STRING_OF_FORK=#{tap.user}
+      -DGMX_INSTALL_LEGACY_API=ON
```

Comment:
Does the legacy API replace the modern API?

Reply:
There is no new API (yet), the main thing the switch does is to install the header files.

Reply:
I'm confused: why would you install this without any headers? It seems very odd for the default build to install things without the headers.

Reply:
> why would you install this without any headers?

Primarily, the project provides a command line tool.

Several libraries with various degrees of support are also bundled. One of them (libgmxapi) is enabled by default and installs `gmxapi/` headers. The oldest (`libgromacs`) is primarily for internal use to support the command line tool(s). Some of its headers are available, optionally, but it is not well-supported for use by external software.